setup.py

```diff
@@ -27,7 +27,7 @@ def run(self):
 PACKAGES = find_packages(where='.', exclude=['dateutil.test'])
 
 def README():
-    with open('README.rst') as f:
+    with open('README.rst', encoding='utf-8') as f:
```

Comment:
I think `open` does not take this keyword in Python 2.7.

`io.open` is the same as Python 3's `open` in both 2.7 and 3. That will fix this.